README.md

```diff
@@ -160,6 +160,11 @@ The map of containers consists of the name of the container mapped to the contai
 * `build` (object, optional): Parameters mapped to Docker's `build`.
 	* `context` (string)
 	* `file` (string)
+* `push` (object, optional): Parameters related to Docker's `push`.
+  * `skip` (boolean): prevents pushing image when `push` is called
```

Comment:
instead of having a flag per container, I would list all the "pushable" containers in a group, allowing me to push everything I want within a single `git push` without any modification to crane.